webextension/scripts/popup.js

```diff
@@ -147,16 +147,27 @@ function makeValidURL() {
 
 function useSearchBoxValue() {
```

Comment:
Would be better if you can make this work by passing in seachValue into the function, rather than relying on a global seachValue.